dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/openapi/ApiFinalise.java

```diff
@@ -565,7 +605,8 @@ private Map<String, List<Api.Endpoint>> groupEndpointsByAbsolutePath() {
       for (String cPath : c.getPaths()) {
         for (Api.Endpoint e : c.getEndpoints()) {
           for (String ePath : e.getPaths()) {
-            String absolutePath = cPath + ePath;
+            String absolutePath =
+                cPath + (cPath.endsWith("/") || ePath.startsWith("/") ? "" : "/") + ePath;
```

Comment:
is it possible that 2 `/` could be concatenated? (`/` from the end + `/` from the start)
is there anything in place to prevent that if it could be an issue?

Reply:
is it possible that 2 `/` could be concatenated? (`/` from the end + `/` from the start)
is there anything in place to prevent that if it could be an issue?

Reply:
Good point. It might be a sping feature as well that they unify `//` from concatenation to only `/`. We might want to do the same.

Reply:
I added a `replace("//","/")` to it to take care of that